index.html

```diff
@@ -3,12 +3,30 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Hello Front-End</title>
+  <title>Elaine's Tic-Tac-Toe</title>
   <link rel="stylesheet" type="text/css" href="css/style.css">
 </head>
 <body>
-  <h1>Hello Front-End</h1>
-
+  <div class="titles">
+  <h1 class="titles" id="intro"></h1>
+  <h1 class="titles" id="second"></h1>
+  <h1 class="titles" id="third"></h1>
+  <h1 class="titles" id="final"></h1>
+</div>
+<div class="gameboard boardGame" id="mainElement">
+<img class="gameboard square" id="topLeft">
+<img class="gameboard square" id="topMiddle">
+<img class="gameboard square" id="topRight">
+<img class="gameboard square" id="middleLeft">
+<img class="gameboard square" id="middleMiddle">
+<img class="gameboard square" id="middleRight">
+<img class="gameboard square" id="bottomLeft">
+<img class="gameboard square" id="bottomMiddle">
+<img class="gameboard square" id="bottomRight">
+</div>
+<div class="results">
+<h3 class="">Click to Begin!</h3>
+</div>
```

Comment:
Hey Elaine - In the future, please be careful with your indentation.  Generally, if you have a child element, it should be tabbed once relative to the parent (so your line 9 is good, line 10 is good, but the h1 on line 11 is contained inside of the div from line 10, so it needs to be tabbed once more).  If you have any questions about how to indent, please reach out to me or talk with a TA!  It helps make code a lot more readable!